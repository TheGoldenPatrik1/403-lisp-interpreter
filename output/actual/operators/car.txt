1
(1 2)
NIL
