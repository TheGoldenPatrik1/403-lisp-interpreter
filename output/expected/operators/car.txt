1
(1 2)
NIL
1